app/views/requests/_hidden_form_fields.html.erb

```diff
@@ -1,6 +1,6 @@
-<%= f.hidden_field :item_id %>
-<%= f.hidden_field :origin %>
-<%= f.hidden_field :origin_location %>
+<%= f.hidden_field :item_id, autocomplete: 'off'  %>
+<%= f.hidden_field :origin, autocomplete: 'off' %>
+<%= f.hidden_field :origin_location, autocomplete: 'off' %>
```

Comment:
Does this `autocomplete` option actually work/cause issues on `input[type="hidden"]`?